idd/Energy+.idd.in

```diff
@@ -24195,6 +24203,164 @@ AirflowNetwork:Distribution:Linkage,
       \note The zone name is where AirflowNetwork:Distribution:Component:Duct is exposed. Leave this field blank if the duct
       \note conduction loss is ignored.
 
+AirflowNetwork:Distribution:DuctViewFactors,
+       \memo This object is used to allow user-defined view factors to be used for duct-surface radiation
+       \memo calculations.
+  A1,  \field Linkage Name
```

Comment:
Is this the name of another object?  If so, need \note to explain and \object-list  Either way should have a \note